content/en/docs/03/_index.md

```diff
@@ -88,7 +88,7 @@ We're using a simple sample application written in Go, which you can find built
 
 {{% param distroName %}} creates the defined and necessary resources, pulls the container image (in this case from Quay.io) and deploys the Pod.
 
-Use the command `{{% param cliToolName %}} get` with the `-w` parameter in order to get the requested resources and afterwards watch for changes.
+Use the command `{{% param cliToolName %}} get` with the `-w` parameter in order to get the requested resources and afterward watch for changes.
```

Comment:
Both versions work.

Reply:
The word ‘informations’ is a legal term. In standard English, the word ‘information’ is a non-count noun.

Reply:
I'm referring to "afterward" and "afterwards". Both work.